client/src/pages/SymptomsCheck.js

```diff
@@ -167,7 +165,7 @@ const Step3 = (props) => {
           key={i}
           text={answer}
           onSelect={() => toggleAnswer(answer)}
-          checked={checked}
```

Comment:
The issue now is if I select this option, I cannot select any other options unless I deselect it. I think we should be able to select other options but uncheck "None of those".

Reply:
@snowiesuet i'm not sure what you mean. Currently, you can select all options and when you select 'None of These', it should unselect all.

Reply:
Yes, after you select "None of those", you should be able to select other options which should deselect "None of those". But currently selecting "None of those" disables other options, I'm not sure if this is a bug or by design?

Reply:
I implemented it based on step 2. Im not sure why they initially started it this way.

Reply:
Ah, I see. I guess this was already there, sorry about the confusion!